x86_64-unknown-intermezzos-gnu.json

```diff
@@ -3,15 +3,20 @@
 	"cpu": "x86-64",
 	"data-layout": "e-m:e-i64:64-f80:128-n8:16:32:64-S128",
 	"executables": true,
-        "linker-flavor": "gcc",
+	"linker": "gcc",
```

Comment:
* [The default linker _(really: compiler)_ binary is `cc`](https://github.com/rust-lang/rust/blob/4cb396c68088f38e517ac8890030d17a969e57ba/src/librustc_back/target/mod.rs#L417)
* [The string seems to be simply used to shell out](https://github.com/rust-lang/rust/blob/4cb396c68088f38e517ac8890030d17a969e57ba/src/librustc_trans/back/write.rs#L1132)